docs/modules/chalk.md

```diff
@@ -0,0 +1,35 @@
+# chalk
+
+`chalk` and similar libraries have various alternatives which are lighter and faster.
+
+# Alternatives
+
+## picocolors
+
+A widely used alternative which is much lighter and faster.
+
+[Project Page](https://github.com/alexeyraspopov/picocolors)
+
+[npm](https://npmjs.com/package/picocolors)
+
+## ansis
+
+A lighter alternative which also supports 256 color creation.
+
+[Project Page](https://github.com/webdiscus/ansis)
+
+[npm](https://npmjs.com/package/ansis)
+
+## styleText (Node 20.x and above)
```

Comment:
considering this is a stable feature that is in the two latest major lts versions, should it be moved up in the list? considering is the smallest (no deps) option :thinking:

Reply:
The important one I think is LTS. If it is in all LTS versions then it's pretty much widely available

Reply:
20.x and 22.x are both lts now :eyes: although 18.x is still supported, just in maintenance

Reply:
Let's leave it for now and we can bump it up once 18 falls out of maintenance (apparently in a few months)